.travis.yml

```diff
@@ -5,4 +5,4 @@ env:
 node_js:
   - '5.4'
 script:
-  - npm run lint
+  - build/script.bash
```

Comment:
I'd favour npm scripts in package json over bash scripts, especially since windows users will probably _not_ have bash and they might want to test the build process locally.